SharedSource/Development/NUnit/UnitTesting/Globalization/AvailableResourcesTester.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+
+// ReSharper disable once CheckNamespace
+namespace Remotion.Development.NUnit.UnitTesting.Globalization
+{
+  public static class AvailableResourcesTester
+  {
+    /// <summary>
+    /// Tests that the available resources declared by <typeparamref name="TAttribute"/> on <paramref name="targetAssembly"/>
+    /// match the resource files in the output directory.
+    /// </summary>
+    public static void CheckAvailableResources<TAttribute> (Assembly targetAssembly)
+      where TAttribute : Attribute
+    {
+      // We don't have a reference to globalization so the call has to provide the right attribute
```

Comment:
waiting on the final decision if linking the attribute into the dev project would make sense or not.

Reply:
Linking does not work since a reference in the XML docs cannot be found. Including this file as well causes more XML doc issues and other compile errors because stuff could not be found.